Support while in pass base

angelayi opened the pull request:
Summary: Fixes https://github.com/pytorch/executorch/issues/8769

Differential Revision: D77382557

lucylq reviewed (approved):
thanks for adding this support!

angelayi committed:
Support while in pass base (#12004)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/12004

Fixes https://github.com/pytorch/executorch/issues/8769

Reviewed By: lucylq

Differential Revision: D77382557
